libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py

```
@@ -128,3 +153,9 @@ def run(test, params, env):
         if cleanup_nfs:
             utils_test.libvirt.setup_or_cleanup_nfs(
                 False, restore_selinux=selinux_bak)
+        if polkit:
+            logging.info("Cleaning up polkit configuration")
+            try:
+                polkit.cleanup()
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)